### PR TITLE
Bug/ con 2960

### PR DESCRIPTION
**Pull request by sijeesh**
Added a new flag "DISABLE_NODE_CONFIGURATION" to disable the node configuration(like package installation,multipath, iscsi,etc.) completely.

### CODE REVIEW

**Review by datamattsson — COMMENTED** [review]
I'm not loving the inconsistency in the expressions we've bestowed upon us in this file.

```
if [ "$arg" = "--node-service" ]; then
if [ "$nodeservice" = true ]; then
if [ "${DISABLE_NODE_CONFORMANCE}" = "true" ]; then
```

Can we also change `configureNode` to `configurenode` for consistency?

**Review by dileepds — APPROVED** [review]

File: cmd/csi-driver/csi-driver.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func csiCliHandler(cmd *cobra.Command) error {
 		flavorName = flavor.Vanilla
 	}
 
-	if nodeService {
+	if nodeService && os.Getenv("DISABLE_NODE_CONFIGURATION") == "false" {
```
[the remark this answers is not in the document]
Comment (sijeesh) [reply cmd/csi-driver/csi-driver.go]: Planning to set a default value for this flag as false. Updated the code to improve readability and consider values other than "true" as false.

**Comment by sijeesh** [comment]
> I'm not loving the inconsistency in the expressions we've bestowed upon us in this file.
> 
> ```
> if [ "$arg" = "--node-service" ]; then
> if [ "$nodeservice" = true ]; then
> if [ "${DISABLE_NODE_CONFORMANCE}" = "true" ]; then
> ```
> 
> Can we also change `configureNode` to `configurenode` for consistency?
Updated entry script to have only camel case variables. Also moved environment variables top to improve readability.

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**